test/unit/SearchResultsPage.test.js

```diff
@@ -27,12 +28,13 @@ const searchResults = {
 const context = mockRouterContext();
 const childContextTypes = {
   router: PropTypes.object,
+  media: PropTypes.string,
 };
 
 describe('SearchResultsPage', () => {
-  describe('Component properties', () => {
-    let component;
+  let component;
```

Comment:
This is weird ... this is saying that component should keep its value through the two sets of tests?  I think it would make more sense for it to be defined multiple times within the describe blocks so it's clear that the tests don't depend on previous values of component from different tests.